### DISCUSSION TITLE
Feature request: adding a "show all resolved" button

### DISCUSSION THREAD

**Discussion by dan3lson** [community/community#9223] [Pull Requests]
When I have a PR open and feedback comes in, I go through each one and eventually mark them as resolved. If someone makes a new comment on a resolved comment, it's not immediately clear where the new comment was made. This results in having to click each accordion-comment container to find where the comment was made.

Two simple improvements can be made from this:
- Have some sort of "Show  all resolved" button (useful when I want to revisit all comments one last time)
- When someone makes a new comment on a resolved comment, automatically set it to be shown so folks don't have to go fishing for it as described above

Thanks! Let me know how I can help.

**Comment by tekumara** [comment]
Alt (or Option) + click on Show/Hide Resolved to unfold/fold them all.
